srcpkgs/broadcom-wl-dkms/template

```diff
@@ -14,6 +14,7 @@ create_wrksrc=yes
 short_desc="Broadcom proprietary wireless drivers for Linux - DKMS kernel module"
 dkms_modules="wl ${version}"
 depends="dkms"
+triggers="dkms initramfs-regenerate"
```

Comment:
This is unnecessary. Almost no users will require that this driver be present in the initramfs, and those that do probably have custom setups to deal with the complexities of bringing up a wireless interface in an early-boot environment anyway. Better to let those users deal with manually updating their images than force every user of this driver to regenerate *all* initramfs images with an update.

Reply:
Removed.